### DISCUSSION TITLE
ORP Protection

### DISCUSSION THREAD

**Discussion by Crom170** [gameserverapp/Platform#728] [Suggestions]
There are plugins that offer ORP protecion like on ASE. you can still be raided while offline but your structures will have more health, and your turrets do more damage. i would love this to GSA as it is much more user friendly,

**Comment by mvaessen** [comment]
Hi, thanks for your suggestion.
There are currently no plans for the future to expand the GSA Integration mod feature set. There might already be plugins or mod that can do this.
